Specs/Scene/TileOrientedBoundingBoxSpec.js

```diff
@@ -28,6 +28,7 @@ defineSuite([
     it('can be instantiated with center and half-axes', function() {
         expect(tileBoundingVolume).toBeDefined();
         expect(tileBoundingVolume.boundingVolume).toBeDefined();
+        expect(tileBoundingVolume.boundingSphere).toBeDefined();
```

Comment:
Can we make this test use more specific expectations for the bounding volume and sphere?